code/modules/synd_specialist/specialist.dm

```diff
@@ -0,0 +1,267 @@
+//The syndicate elite
+//I have no business making something this advanced
+
+
+//Trying to keep it all in one file
+/*----------------
+     CLOTHING
+----------------*/
+
+/obj/item/clothing/suit/agent_coat//the coat
+	name = "Trenchcoat"
+	desc = "A fashionable black leather trenchcoat. Supposedly inconspicuous."
+	allowed = /obj/item/weapon/gun //They can only carry guns in their coat
+	icon_state = "hos"
+	item_state = "greatcoat"
+	slowdown = 0
+	unacidable = 1
+	armor = list(melee = 85, bullet = 70, laser = 70, energy = 0, bomb = 30, bio = 0, rad = 0)
+	strip_delay = INFINITY//No, you can't have it
+
+/obj/item/clothing/under/syndicate/agent_suit
+	name = "Black Ops suit"
+	desc = "Nanotrasen sure pissed off somebody important."
+	strip_delay = INFINITY
+
+
+/*----------------
+    ABILITIES
+----------------*/
+/obj/effect/proc_holder/spell/proc/agent_check(var/mob/living/carbon/human/H)//shamelessly stolen from sling
+	if(!H || !istype(H)) return
+	if(H.dna.species.id == "agent") return 1
+	if(!H.dna.species.id== "agent") usr << "You aren't augmented enough to do this."
+	return 0
+//Make gangs
+/obj/effect/proc_holder/spell/targeted/gang //Opens a selected airlock
+	name = "Recruit"
+	desc = "Brainwashes a target using your augmented glare to start up a syndicate gang."
+	panel = "Augmentation"
+	charge_max = 500
+	clothes_req = 0
+	action_icon_state = "mech_overload_off"
+
+
+/obj/effect/proc_holder/spell/targeted/gang/cast(list/targets,mob/user = usr)
+	for(var/mob/living/target in targets)
+		if(!ishuman(target))
+			user << "<span class='warning'>You may only brainwash humans!</span>"
+			revert_cast()
+			return
+		if(locate(/obj/item/weapon/implant/loyalty) in target)
+			user << "<span class='warning'>[target]'s Nanotrasen microchip prevents brainwashing.</span>"
+		if(target.stat)
+			user << "<span class='warning'>[target] must be conscious!</span>"
+			revert_cast()
+			return
+		var/mob/living/carbon/human/M = target
+		user.visible_message("<span class='warning'><b>[user] locks [target] in their gaze!</b></span>")
+		target.visible_message("<span class='danger'>[target] looks like they realized a revelation!</span>")
+		if(in_range(target, user))
+			target << "<span class='userdanger'>Your gaze is forcibly drawn into [user]'s augmented eyes and you are filled with hatred.</span>"
+		else //Only alludes to the shadowling if the target is close by
```

Comment:
Friendly warning that maintainers tend NOT to like that. Copy-pasta is one of the most hated things in coderbus. Chances are, one of them will require you to change it.